nav.adoc

```diff
@@ -1,3 +1,4 @@
+include::./partials/attributes.adoc[]
```

Comment:
Let's keep `nav.adoc` pure. It's used in a script and I'd prefer not to have attributes in it.